src/core/evaluator.js

```diff
@@ -1606,9 +1616,11 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
               var name = args[0].name;
               if (xobjsCache.key === name) {
-                if (xobjsCache.texts) {
-                  Util.appendToArray(textContent.items, xobjsCache.texts.items);
-                  Util.extendObj(textContent.styles, xobjsCache.texts.styles);
+                xobjsCache.cachedChunks = sink.cachedChunks;
```

Comment:
this line belong below after self.getTextContent call as `xobjsCache.cachedChunks = sinkWrapper.cachedChunks;`